pandas/tests/io/json/test_normalize.py

```diff
@@ -692,3 +691,12 @@ def test_with_large_max_level(self):
         ]
         output = nested_to_record(input_data, max_level=max_level)
         assert output == expected
+
+    def test_deprecated_import(self):
+        with tm.assert_produces_warning(FutureWarning) as w:
+            from pandas.io.json import json_normalize
+            recs = [
```

Comment:
Is this on down necessary? I think should just throw warning on import not on invocation

Reply:
I am not sure about how to deprecate while importing, do you have an idea?